ng2-components/ng2-activiti-form/src/components/widgets/widget.component.ts

```diff
@@ -63,6 +65,20 @@ export class WidgetComponent implements AfterViewInit {
         return false;
     }
 
+    setupMaterialTextField(elementRef: ElementRef, handler: any, value: string): boolean {
+        if (elementRef && handler) {
+            let el = elementRef.nativeElement;
```

Comment:
Is there a benefit to using Angular element querying using `By` instead of `nativeElement.querySelector()` method?